manifests/dynflow/worker.pp

```diff
@@ -51,6 +51,8 @@
       enable    => true,
       subscribe => Class['foreman::database'],
     }
+
+    Anchor <| title == 'postgresql::server::service::end' |> ~> Service[$service]
```

Comment:
The chaining certainly makes sense but I'm not sure this would really notify since I don't see a `Service[postgresqld] ~> Anchor[postgresql::server::service::end]`. Have you verified this?

And spec testing here is unreliable (we've seen that in the past in https://github.com/rodjek/rspec-puppet/pull/821), but you could try to see what this test shows. If it fails, you know for certain it isn't there ;)
```ruby
it { is_expected.to contain_service('postgresqld').that_notifies('Service[dynflow-sidekiq@SOMETHING]') }
```

Reply:
Something something theory and practice ...

What may happen is that something sends a notification to `Class[Postgresql::Serverver::Service]` so all resources within the class are refreshed. I wonder what happens if you run:
```
foreman-installer
systemctl stop postgresql
foreman-installer
```

Does it then also restart Dynflow?

Reply:
Yes, that validates with both rspec and the real world, nice.